code/game/gamemodes/vampire/vampire_datum.dm

```diff
@@ -26,11 +31,13 @@
 
 	if (power.isVerb && power.verbpath)
 		vampire.current.verbs += power.verbpath
-	if (announce)
-		to_chat(vampire.current, "<span class='notice'><b>You have unlocked a new power:</b> [power.name].</span>")
-		to_chat(vampire.current, "<span class='notice'>[power.desc]</span>")
+	if(announce)
+		to_chat(vampire.current, SPAN_NOTICE("------------------"))
+		to_chat(vampire.current, SPAN_NOTICE("<b>You have unlocked a new power:</b> [power.name]."))
+		to_chat(vampire.current, SPAN_NOTICE(" - [power.desc]"))
 		if (power.helptext)
-			to_chat(vampire.current, "<span class='good'>[power.helptext]</span>")
+			to_chat(vampire.current, SPAN_GOOD(" - [power.helptext]"))
+		to_chat(vampire.current, SPAN_NOTICE("------------------"))
```

Comment:
Since Goonchat isn't going anywhere we can use some cool CSS to make stuff even more prominent (and pretty).

![image](https://user-images.githubusercontent.com/781546/98580871-f4bca100-22c0-11eb-8c8a-badf76f77253.png)

Though I should probably make some helpers before mindlessly plopping random CSS everywhere.... Never mind.

Reply:
that looks great, but i think we should make a new PR with all the prettifying stuff

Reply:
Yeah, I also found out that for some reason putting `style="padding: ..."` in there crashes the client so ... ¯\_(ツ)_/¯